fastlane/actions/gitlab_create_project.rb

```diff
@@ -10,13 +10,17 @@ def self.run(params)
         source = Gitlab.client(endpoint: params[:endpoint_src], private_token: params[:api_token_src])
         destination = Gitlab.client(endpoint: params[:endpoint_dst], private_token: params[:api_token_dst])
         original_project = params[:project]
+        if original_project.owner
```

Comment:
It seems this may be the case when a project is created within a *group*.


For debugging:

```
 if original_project.owner
     owner_id = original_project.owner.id
else
    owner_id = -1
end

UI.message ("Creating Project: #{original_project.path_with_namespace}  with owner id #{owner_id}") 
```

then call

```
fastlane migrate project:jupedsim/test
```

gives 

<img width="1140" alt="screen shot 2016-12-17 at 12 42 43" src="https://cloud.githubusercontent.com/assets/5772973/21286521/bae22486-c456-11e6-83e4-d390b1338308.png">

This is the [test](https://cst.version.fz-juelich.de/jupedsim/test) project